File: CoreJavaDay2Assignment/Car.java
package com.sonata;

public class Car {
	int speed;
	double regularPrice;
	String color;
	Car(int speed,double regularPrice,String color){
		this.speed=speed;
		this.regularPrice=regularPrice;
		this.color=color;
	}
	public double getSalePrice()
	{
		return regularPrice;
	}
	public static void main(String args[])
	{
		Car c=new Car(250,1400000,"white");
		System.out.println(c.getSalePrice());
	}

}

File: Typescript_Assignment/Car.ts
export class Car {
    speed: number;
    regularprice: number;
    color: string;

    constructor(s: number, r: number, c: string) {
        this.speed = s;
        this.regularprice = r;
        this.color = c;
    }

    getSalePrice(): number {
        return this.regularprice;
    }
}
/*var car = new Car(140, 1600000, "White");
console.log(car.getSalePrice());*/